packages/react-native-editor/__device-tests__/gutenberg-editor-block-insertion-@canary.test.js

```diff
@@ -60,10 +60,6 @@ describe( 'Gutenberg Editor tests for Block insertion', () => {
 		await editorPage.sendTextToParagraphBlock( 1, testData.longText );
 		// Should have 3 paragraph blocks at this point.
 
-		if ( isAndroid() ) {
-			await editorPage.dismissKeyboard();
```

Comment:
This is now handled within `dismissKeyboard` if the orientation is `LANDSCAPE` (on Android) it will hide the Keyboard using the driver's `hideDeviceKeyboard` method.